README.md

```diff
@@ -37,48 +37,50 @@ on the host system, the minimum build requirements:
 - bc
 
 ## building
-set `DEFCONFIG` to one of the [supported boards](#supported-boards), `DEFCONFIG=rpi3` for example.
-first, configure
+set `DEFCONFIG` to one of the [supported boards](#supported-boards),
 
 ```sh
-. env.sh
-make --directory=$(pwd)/buildroot O=$(pwd)/${DEFCONFIG}_output ${DEFCONFIG}_defconfig
+$ export DEFCONFIG=rpi4
```

Comment:
I think this needs to be raspberrrypi4 not rpi4

Reply:
i've been using `rpi4` for months

Reply:
i've been using `rpi4` for months
maybe the stock name is `raspberrypi4`?

Reply:
this follows the names from `accessibility-oscilloscope/configs/${DEFCONFIG}_defconfig`, so [this](https://github.com/accessibility-oscilloscope/buildroot/blob/buildroot-build-script/accessibility-oscilloscope/configs/rpi4_defconfig)

raspberrypi4 iirc is the default config from buildroot, so i wouldn't be surprised if that was working and ignoring the project-specific configs

Reply:
yeah, with `DEFCONFIG=raspberrypi4`

```sh
$ ls buildroot/configs/ | rg raspberrypi
raspberrypi0_defconfig
raspberrypi0w_defconfig
raspberrypi2_defconfig
raspberrypi3_64_defconfig
raspberrypi3_defconfig
raspberrypi3_qt5we_defconfig
raspberrypi4_64_defconfig
raspberrypi4_defconfig
raspberrypi_defconfig
```

that's pulling in the default config provided by buildroot. we based `rpi4_defconfig` off of this file and it's probably not that much different still, so makes sense it would be easy to miss.